docs/src/main/sphinx/connector/atop.rst

```diff
@@ -0,0 +1,182 @@
+==============
+Atop connector
+==============
+
+The Atop connector supports reading disk utilization statistics from the ``atop``
+(Advanced System and Process Monitor) Linux server performance analysis tool.
+
+Requirements
+------------
+
+In order to use this connector, the host on which the Trino worker is running
+needs to have the ``atop`` tool installed locally.
+
+Connector configuration
+-----------------------
+
+The connector can read disk utilization statistics on the Trino cluster.
+Create a catalog properties file that specifies the Atop connector by
+setting the ``connector.name`` to ``atop``.
+
+For example, create the file ``etc/catalog/system_monitor.properties``
+and replace the connector properties as appropriate for your setup:
+
+.. code-block:: text
+
+    connector.name=atop
+    atop.executable-path=/usr/bin/atop
+
+Configuration properties
+------------------------
+
+.. list-table:: Configuration variables
+  :widths: 40, 10, 10, 40
+  :header-rows: 1
+
+  * - Property name
+    - Default value
+    - Required
+    - Description
+  * - ``atop.concurrent-readers-per-node``
+    - ``1``
+    - Yes
+    - The number of concurrent read operations allowed per node.
+  * - ``atop.executable-path``
+    - (none)
```

Comment:
@mosabua We should have a convention for documenting values which don't have default values (and document this) for consistency.

No change requested @findinpath .